### PR TITLE
REL: 2.4.0

### PR DESCRIPTION
**Pull request by effigies**
Preparation for nibabel release 2.4.0, targeting Monday, 25 March 2019.

The [2.4.0 milestone](https://github.com/nipy/nibabel/milestone/2) has been added for issues that should be addressed before a minor version release or pull requests that should be merged or deferred. Please add any issues or pull requests that need attention to the milestone or comment in this thread.

Issues/PRs that need to be addressed, merged or postponed
---------------------
* [x] #567 [PR] (Submitter: @effigies; Reviewers: @matthew-brett @yarikoptic)
* [x] #641 [PR] (Submitter: @MichielCottaar; Reviewer: @demianw)
* [x] #644 [PR] (Submitter: @effigies; Reviewer: @matthew-brett)
* [x] #652 [PR] (Submitter: @mattcieslak; Reviewer: @effigies)
* [x] #688 [PR] (Submitter: @takhs91; Reviewer: @effigies)
* [x] #691 [BUG] (Submitter: @ashgillman)
* [x] #692 [PR] (Submitter: @Roosted7; Reviewer: @effigies)
* [x] #727 [PR] (Submitter: @mgxd; Reviewer: @effigies)
* [x] #728 [PR] (Submitter: @reddigari; Reviewer: @effigies)
* [x] #736 [PR] (Submitter: @effigies)
* [x] #741 [PR] (Submitter: @effigies, fixing #716 [BUG] by @constracti)

This is a partial list of the open PRs, based on my judgment of how close they are to ready or how likely the submitter is to still be active. If you think I've missed some, please let me know. I will try to update all of these issues/PRs, even if they don't get resolved this round.

Submitters: I would appreciate if all PRs could be merged or rebased with the current master.

Additional help pushing forward these issues or reviewing PRs would be very welcome.

I will merge #736 before releasing in the event of no reviews.

Pre-release checklist
--------
* [x] Review the open list of [nibabel issues](https://github.com/nipy/nibabel/issues). Check whether there are outstanding issues that can be closed, and whether there are any issues that should delay the release. Label them!
* [x] Review and update the release notes. Review and update the [Changelog file](https://github.com/nipy/nibabel/blob/master/Changelog).
* [x] Look at [`doc/source/index.rst`](https://github.com/nipy/nibabel/blob/master/doc/source/index.rst) and add any authors not yet acknowledged.
* [x] Use the opportunity to update the [`.mailmap file`](https://github.com/nipy/nibabel/blob/master/.mailmap) if there are any duplicate authors listed from `git shortlog -nse`.
* [x] Check the copyright year in [doc/source/conf.py](https://github.com/nipy/nibabel/blob/master/doc/source/conf.py)
* [x] Refresh the [README.rst](https://github.com/nipy/nibabel/blob/master/README.rst) text from the `LONG_DESCRIPTION` in [`info.py`](https://github.com/nipy/nibabel/blob/master/nibabel/info.py) by running `make refresh-readme`.
* [x] Check the dependencies listed in [`nibabel/info.py`](https://github.com/nipy/nibabel/blob/master/nibabel/info.py) (e.g. `NUMPY_MIN_VERSION`) and in [`doc/source/installation.rst`](https://github.com/nipy/nibabel/blob/master/doc/source/installation.rst) and in [`requirements.txt`](https://github.com/nipy/nibabel/blob/master/requirements.txt) and [`.travis.yml`](https://github.com/nipy/nibabel/blob/master/.travis.yml). They should at least match. Do they still hold? Make sure nibabel on travis is testing the minimum dependencies specifically.
* [x] Make sure all tests pass (from the nibabel root directory): `nosetests --with-doctest nibabel`
* [x] Edit [`nibabel/info.py`](https://github.com/nipy/nibabel/blob/master/nibabel/info.py) to set `_version_extra` to `''`; commit

I'm skipping the build-bots, as they don't seem to buy us much that isn't in the CI. @matthew-brett Please let me know if there's a good reason to keep using them.

Adapted from http://nipy.org/nibabel/devel/make_release.html#release-checklist

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -162,6 +162,9 @@
     {
       "name": "Nguyen, Ly"
     },
+    {
+      "name": "Reddigari, Samir"
```
Comment (effigies) [inline .zenodo.json]: @reddigari Let me know if you have another name you prefer for publications, your affiliation, and your [ORCID](https://orcid.org/).

Comment (reddigari) [reply .zenodo.json]: Affiliation: BrainSpec, Boston, MA
ORCID: 0000-0003-1472-5881

File: .zenodo.json [hunk 2]
```diff
@@ -207,6 +212,9 @@
       "name": "Jordan, Kesshi",
       "orcid": "0000-0001-6313-0580"
     },
+    {
+      "name": "Cieslak, Matt"
```
Comment (effigies) [inline .zenodo.json]: @mattcieslak Let me know if you have another name you prefer for publications, your affiliation, and your ORCID. (Sorry if you've given this to me in another package, but I couldn't find it from a quick look.)

Comment (mattcieslak) [reply .zenodo.json]: Here it is @effigies 
```
    {
      "name": "Cieslak, Matthew",
      "affiliation": "Department of Neuropsychiatry, University of Pennsylvania",
      "orcid": "0000-0002-1931-4734"
    },
```

Comment (effigies) [reply .zenodo.json]: Gah, just a few minutes too late! I'll have it in for the 2.4.1 release.

**Pull request merged**